### ISSUE TITLE
Skills multi URL support

### ISSUE THREAD

**Issue by gmartin50** [chamilo/chamilo-lms#3331]
Hi I'm noticing that Skills that I create in one URL are visible in all others. Is this by design?

Thanks for your response

**Comment by ywarnier** [comment]
Yes, there is no plan at this point to have different skills trees between different URLs.
In my opinion, if you need that, you're probably better off using different Chamilo installations, but I'm open to hearing your use case...
